src/rootfinding.jl

```diff
@@ -206,3 +206,145 @@ function taylorinteg(f!, g, q0::Array{U,1}, t0::T, tmax::T,
 
     return view(tv,1:nsteps), view(transpose(view(xv,:,1:nsteps)),1:nsteps,:), view(tvS,1:nevents-1), view(transpose(view(xvS,:,1:nevents-1)),1:nevents-1,:), view(gvS,1:nevents-1)
 end
+
+"""
+    taylorinteg(f, g, x0, trange, order, abstol; kwargs... )
+
+Root-finding method of `taylorinteg`; returns the solution of the ODE evaluated
+*only* at the times specified by `trange`. Given a function `g(t, x, dx)`,
+called the event function, `taylorinteg` checks for the occurrence of a root
+of `g` evaluated at the solution; that is, it checks for the occurrence of an
+event or condition specified by `g=0`. Then, `taylorinteg` attempts to find that
+root (or event, or crossing) by performing a Newton-Raphson process. When
+called with the `eventorder=n` keyword argument, `taylorinteg` searches for the
+roots of the `n`-th derivative of `g`, which is computed via automatic
```

Comment:
Question: I guess that `eventorder` should satisfy `order >= eventorder`, i.e., it should be smaller than the order of series expansion used to approximate the solution. Is that checked somewhere?

Reply:
Yes, `eventorder` should satisfy `order >= eventorder` but currently that is not checked, I will add an assertion, many thanks for noticing this!

Reply:
Just added some `@asserts` which check this condition, thanks again!

Reply:
Thank's a lot, I think this is solved.